src/main/java/com/replyyes/facebook/messenger/bean/FacebookMessengerSendException.java

```diff
@@ -0,0 +1,18 @@
+package com.replyyes.facebook.messenger.bean;
+
+import lombok.Getter;
+
+@Getter
+public class FacebookMessengerSendException extends Exception {
```

Comment:
Well, this 100% needs to be a new exception in order to pass the required payload to the caller. This _could_ extend `IOException`, but I don't particularly feel that this comes under the purview of an input/output error.